ui/src/actions/literature.js

```diff
@@ -154,11 +156,16 @@ export function assignPapers(conferenceId, conferenceTitle) {
 }
 
 export function exportToCds() {
-  return (dispatch, getState, http) => {
-    const papers = getState().literature.get('literatureSelection');
-    http.post('/assign/export-to-cds', {
-      literature_recids: papers,
-    });
-    dispatch(clearLiteratureSelection());
+  return async (dispatch, getState, http) => {
+    try {
+      const papers = getState().literature.get('literatureSelection');
```

Comment:
we're missing the `loading` action here, we should create a task for that.

Reply:
@drjova I can add it, it shouldn't take long

Reply:
That would be great, thanks!